frontend/components/base/base-render-markdown/BaseRenderMarkdown.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="markdown-render" v-html="markdownToHtml" />
+</template>
+<script>
+import { marked } from "marked";
+import { markedHighlight } from "marked-highlight";
+import hljs from "highlight.js";
+import * as DOMPurify from "dompurify";
+marked.use(
+  markedHighlight({
+    langPrefix: "hljs language-",
+    highlight(code, lang) {
+      const language = hljs.getLanguage(lang) ? lang : "plaintext";
+      return hljs.highlight(code, { language }).value;
+    },
+  })
+);
+export default {
+  props: {
+    markdown: {
+      type: String,
+      required: true,
+    },
+  },
+  computed: {
+    markdownToHtml() {
+      const parsed = marked.parse(this.markdown, {
+        headerIds: false,
+        mangle: false,
+      });
+      return DOMPurify.sanitize(parsed);
+    },
+  },
```

Comment:
since `markdownToHtml` is a reactive computed, it will be calculate each time his dependencies change.
In this case, it's depend only in one the props `this.markdown`.

=> If the value doesn't need to be reactive, use a static variable instead of a computed